charts/scalardl/Chart.yaml

```diff
@@ -12,7 +12,8 @@ keywords:
 - ledger
 - dlt
 home: https://scalar-labs.com/
-source: ["https://scalardl.readthedocs.io/en/latest/"]
+sources:
+  - https://scalardl.readthedocs.io/en/latest
```

Comment:
```suggestion
  - https://github.com/scalar-labs/scalardl
```

Reply:
Thank you for your suggestion.
Source URL updated 9f761b07464ca50ad3e370c84b66e99b6d3faeae